psydac/api/discretization.py

```diff
@@ -504,7 +504,7 @@ def discretize_space(V, domain_h, *, degree=None, multiplicity=None, knots=None,
 
     construct_reduced_interface_spaces(g_spaces, new_g_spaces, interiors, connectivity)
 
-    Vh = ProductFemSpace(*new_g_spaces.values(), connectivity=connectivity)
+    Vh = create_product_space(*new_g_spaces.values(), connectivity=connectivity)
```

Comment:
Is this the only place in the library where the factory function `create_product_space` is used?

Reply:
Apart from the dubious concrete `__mul__` and `__rmul__` functions in the abstract FemSpace, yes.

Reply:
(they were also used in some tests, but as you noticed they could be replaced by a plain VectorFemSpace constructor)

Reply:
I see. Then please use the class constructor here if possible

Reply:
I just removed the factory function